test/integration/root_overview_test.rb

```diff
@@ -25,4 +25,282 @@ class RootOverviewTest < IntegrationTest
+      :guide_edition,
+      title: "ZZZ",
+      slug: "zzz",
+      state: "in_review",
+      review_requested_at: 20.minutes.ago,
+    )
+
+    visit "/"
+    filter_by_user("All")
+    filter_by_status("In review")
+
+    find(".publications-table tr:nth-child(1) details").click
+    assert page.has_css?(".publications-table tr:nth-child(1) details .govuk-summary-list__row:nth-child(3) .govuk-summary-list__value", text: "20 minutes")
+
+    find(".publications-table tr:nth-child(2) details").click
+    assert page.has_css?(".publications-table tr:nth-child(2) details .govuk-summary-list__row:nth-child(3) .govuk-summary-list__value", text: "2 days")
+
+    find(".publications-table tr:nth-child(3) details").click
+    assert page.has_css?(".publications-table tr:nth-child(3) details .govuk-summary-list__row:nth-child(3) .govuk-summary-list__value", text: "4 days")
+  end
+
+  should "allow a user to claim 2i" do
+    stub_linkables
+    stub_holidays_used_by_fact_check
+
+    user = FactoryBot.create(:user, :govuk_editor)
+    assignee = FactoryBot.create(:user, :govuk_editor)
+    edition = FactoryBot.create(
+      :guide_edition,
+      title: "XXX",
+      state: "in_review",
+      review_requested_at: Time.zone.now,
+      assigned_to: assignee,
+    )
+
+    visit "/"
+    filter_by_user("All")
+    filter_by_status("In review")
+    find(".publications-table tr:first-child details").click
+
+    within(".publications-table tr:first-child details .govuk-summary-list__row:nth-child(4) .govuk-summary-list__value") do
+      find_button("Claim 2i").click
+    end
+
+    assert edition_url(edition), current_url
+    assert page.has_content?("You are the reviewer of this guide.")
+    assert page.has_select?("Reviewer", selected: user.name)
+    assert page.has_select?("Assigned to", selected: assignee.name)
+  end
+
+  should "prevent claiming 2i when someone else has" do
+    stub_linkables
+    stub_holidays_used_by_fact_check
+
+    FactoryBot.create(:user, :govuk_editor)
+
+    assignee = FactoryBot.create(:user, :govuk_editor)
+    another_user = FactoryBot.create(:user, :govuk_editor, name: "Another McPerson")
+    edition = FactoryBot.create(
+      :guide_edition,
+      title: "XXX",
+      state: "in_review",
+      review_requested_at: Time.zone.now,
+      assigned_to: assignee,
+    )
+
+    visit "/"
+    filter_by_user("All")
+    filter_by_status("In review")
+    find(".publications-table tr:first-child details").click
+
+    edition.reviewer = another_user.name
+    edition.save!
+
+    within(".publications-table tr:first-child details .govuk-summary-list__row:nth-child(4) .govuk-summary-list__value") do
+      find_button("Claim 2i").click
+    end
+
+    assert edition_url(edition), current_url
+    assert page.has_content?("Another McPerson has already claimed this 2i")
+    assert page.has_select?("Reviewer", selected: another_user.name)
+    assert page.has_select?("Assigned to", selected: assignee.name)
+
```

Comment:
once we have checked that one cant claim 2i when someone else has, I dont know if there is a need to test the remaining of the test? or may be if we want to do that we can express the intention in the test.

Reply:
Yes, good point. I've kept the test as is but updated the description: I think it is a useful thing to test for still.